packages/actor-scraper/web-scraper/src/internals/bundle.browser.ts

```diff
@@ -0,0 +1,235 @@
+/* eslint-disable max-classes-per-file */
+/* istanbul ignore next */
+
+// TODO: properly type this file, mostly copy paste from context.ts in scraper-tools
```

Comment:
i believe this file was done in a way so we dont need to compile it anyhow. if we want to have it in TS, we might need some additional things to build it properly (all packages are for node 15+, we cant apply the same constraints to this file as its for browsers)